diff --git a/tests/behat/features/capabilities/embed/embed.feature b/tests/behat/features/capabilities/embed/embed.feature
index ea5c1d819..e4d6511ed 100644
--- a/tests/behat/features/capabilities/embed/embed.feature
+++ b/tests/behat/features/capabilities/embed/embed.feature
@@ -19,7 +19,7 @@ Feature: Embed
     And I click on the embed icon in the WYSIWYG editor
     And I wait for AJAX to finish
     And I fill in "URL" with "https://www.youtube.com/watch?v=kgE9QNX8f3c"
-    And I click the xth "0" element with the css ".url-select-dialog .form-actions .ui-button"
+    And I click the xth "0" element with the css ".url-select-dialog .form-actions .form-submit"
     And I wait for AJAX to finish
     And I wait for "3" seconds
     And I press "Save"