diff --git a/modules/custom/activity_basics/activity_basics.module b/modules/custom/activity_basics/activity_basics.module
index f4942e285..693765fcd 100644
--- a/modules/custom/activity_basics/activity_basics.module
+++ b/modules/custom/activity_basics/activity_basics.module
@@ -13,9 +13,28 @@ use Drupal\group\Entity\GroupContent;
  * Implements hook_entity_insert().
  */
 function activity_basics_entity_insert(EntityInterface $entity) {
+  $source_entity = $entity;
+
+  if ($source_entity->getEntityTypeId() == 'group_content') {
+    $source_entity = $source_entity->getEntity();
+  }
+
+  if ($source_entity->getEntityTypeId() == 'node' && !$source_entity->isPublished()) {
+    return;
+  }
+
   _activity_basics_entity_action($entity, 'create_entitiy_action');
 }

+/**
+ * Implements hook_ENTITY_TYPE_update().
+ */
+function activity_basics_node_update(EntityInterface $entity) {
+  if (!$entity->original->isPublished() && $entity->isPublished()) {
+    _activity_basics_entity_action($entity, 'create_entitiy_action');
+  }
+}
+
 /**
  * Implements hook_social_group_move().
  */
